docs/getting_started.md

```diff
@@ -25,12 +25,12 @@ Before starting Prometheus, let's configure it.
 
 ## Configuring Prometheus to monitor itself
 
-Prometheus collects metrics from monitored targets by scraping metrics HTTP
-endpoints on these targets. Since Prometheus also exposes data in the same
+Prometheus collects metrics from _targets_ by scraping `metrics` HTTP
```

Comment:
metrics is not a literal here, so it shouldn't be in code tags

Reply:
Removing.  I'd interpreted this as shorthand for `http://localhost:9100/metrics`, though it could go either way.